packages/wonder-blocks-button/generated-snapshot.test.js

```diff
@@ -6,6 +6,9 @@
 //   2. Run `yarn run gen-snapshot-tests`.
 import React from "react";
 import renderer from "react-test-renderer";
+
+// Mock react-dom as jest doesn't like findDOMNode.
+jest.mock("react-dom");
```

Comment:
I'm moving this general change into a different PR, but needed it here to get tests to pass.